css/main.css

```diff
@@ -559,4 +553,120 @@ textarea {
     width: 3vw;
     height: 32px;
   }
-}
\ No newline at end of file
+}
+
+/* Mobile Menu styling */
+
+.header {
+  background-color: #3c3a39;
+  color: #fff5e1;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem 1.5rem;
+}
```

Comment:
- [ ] Good job using `flex` to position elements but setting `justify-content` and `align-items` to `flex-start` can help you position the nav-links as shown in the requirements. Also please consider referring [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) for more on positioning elements using flex.